quickwit-metastore/src/metastore/mod.rs

```diff
@@ -148,6 +151,7 @@ impl IndexMetadata {
             search_settings,
             sources: Vec::new(),
             create_timestamp: Utc::now().timestamp(),
+            update_timestamp: Utc::now().timestamp(),
```

Comment:
nitpick: Can we call now once and use the same value on create_timestamp and update_timestamp?
It could be nice for debugging to be able to identify index htat have never been modified.

(I know we can't have that sweet logic on the default of deserialization on older version but that's ok)